src/ui/index.css

```diff
@@ -3,11 +3,11 @@
 body, html {
     margin: 0;
     padding: 0;
-    font-family: Arial, sans-serif; 
+    font-family: "Segoe UI", Tahoma, Verdana, sans-serif;
 }
 
 .navbar {
-    background-color: #382459;
+    background-color: #44144c;
```

Comment:
In a followup we should clean up the CSS to use constants for a lot of these, but we're in the prototype phase so lets not worry now.